src/main/java/chess/service/ChessWebService.java

```diff
@@ -0,0 +1,78 @@
+package chess.service;
+
+import chess.dao.HistoryDao;
+import chess.domain.game.ChessGame;
+import chess.domain.game.NormalStatus;
+import chess.domain.position.MovingPosition;
+import chess.dto.*;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class ChessWebService {
+	public void clearHistory() throws SQLException {
+		HistoryDao historyDao = new HistoryDao();
+
+		historyDao.clear();
+	}
+
+	public ChessGameDto setBoard() throws SQLException {
+		ChessGame chessGame = new ChessGame();
```

Comment:
```suggestion
		ChessGame chessGame = load();
```
이렇게 만들면 체스게임 생성하는 중복 코드를 제거할 수 있을거에요